src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+import duke.Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public static int taskNumber = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
It's good that the comment explains what the unicodes represent, but perhaps it would be more elegant to replace them with your own user-declared constants?